py/runai_model_streamer/runai_model_streamer/file_streamer/file_streamer.py

```diff
@@ -12,14 +12,35 @@
 from runai_model_streamer.file_streamer.requests_iterator import (
     RequestsIterator,
 )
+
+from runai_model_streamer.s3_utils.s3_utils import (
+    S3Credentials,
+    is_s3_path,
+    is_gs_path,
+    set_gs_environment_variables,
+    convert_gs_path,
+)
+
 import humanize
+import importlib
 
+s3_module = importlib.util.find_spec("runai_model_streamer_s3") if importlib.util.find_spec("runai_model_streamer_s3") else None
+s3_credentials_module_name = "runai_model_streamer_s3.credentials.credentials"
+if s3_module is not None and importlib.util.find_spec(s3_credentials_module_name):
+    s3_credentials_module = importlib.import_module(s3_credentials_module_name)
+else:
+    s3_credentials_module = None
+
+# use env variable to not use boto3
```

Comment:
I dont think this comment need to be in this place. Its not what people who come to look at FileStreamer class are need to know.

Maybe above line 42